lite/helpers.go

```diff
@@ -71,20 +71,25 @@ func (pkz privKeys) ToValidators(init, inc int64) *types.ValidatorSet {
 
 // signHeader properly signs the header with all keys from first to last exclusive.
 func (pkz privKeys) signHeader(header *types.Header, first, last int) *types.Commit {
-	votes := make([]*types.Vote, len(pkz))
+	cs := make([]*types.CommitSig, len(pkz))
```

Comment:
`cs` is a common name for consensus. Can I suggest a different name? `sigs` or `commitSigs` or `precommits` perhaps

Reply:
:+1: